doc/METADATA.ini

```diff
@@ -371,6 +371,15 @@ description= "introduces a sub-command
 
 	must be specified on a key in the 'spec' namespace"
 
+[command/key]
+type= string
+status= implemented
+usedby/plugin= gopts
+usedby/api= elektraGetOpts kdbopts.h
+description= "specifies which (sub-)command a key in the
+
+			 'spec' namespace belongs to"
+
```

Comment:
`command/key` shouldn't be listed here. It is only used internal. It isn't set by the user or returned to the caller, so we shouldn't document it here.

Reply:
Thanks, I wasn't aware of that.